src/components/InstantSearchSsr.vue

```diff
@@ -0,0 +1,10 @@
+<template>
+  <!-- AisSsr is an empty component that will hold other widgets -->
```

Comment:
```suggestion
  <!-- AisInstantSearchSsr is an empty component that will hold other widgets -->
```

Reply:
This comment is removed with the .js file